package.json

```diff
@@ -11,6 +11,10 @@
     "es",
     "assets/*.css"
   ],
+  "sideEffects": [
+    "es/**/style/*",
+    "lib/**/style/*"
```

Comment:
`assets` dir contains component styles, should add `assets/*`

Reply:
👌
```suggestion
    "lib/**/style/*",
    "assets/*"
```